packages/core/src/types/IViewport.ts

```diff
@@ -69,6 +70,11 @@ export type ReferenceCompatibleOptions = {
    * not need to be provided.
    */
   imageURI?: string;
+
+  /**
+   * To see if the reference could be overladed (labelmap, fusion) on the viewport, set this to true.
```

Comment:
overladed => overlayed
But, as I mentioned, I think it is putting too much into the one function as it changes the signature of it and adds a lot of getter code.